### PR TITLE
TAU miniaod validation/dqm

### PR DESCRIPTION
**Pull request by aspiezia**
#### PR description:
This PR has been done in order to add some histograms for the tau validation. The histograms are meant to validate the miniAOD part, in particolar for what it concern the DeepTau discriminators that can be checked only at miniAOD level. @rmanzoni @roger-wolf 

#### PR validation:
First the changes have been tested with some RelVal samples in order to check that the histograms are created and filled in a sensible way.
Then, the instructions of https://cms-sw.github.io/PRWorkflow.html (see [*]) have been followed.

[*]
cd src
scram b distclean 
git cms-checkdeps -a -A
scram b -j 8
scram b runtests
runTheMatrix.py -l limited -i all --ibeos

### CODE REVIEW

**Comment by kpedro88** [comment]
please test

File: Configuration/StandardSequences/python/Validation_cff.py [hunk 1]
```diff
@@ -63,7 +64,7 @@
 validationLiteTracking.replace(globalValidation,globalValidationLiteTracking)
 validationLiteTracking.remove(condDataValidation)
 
-validationMiniAOD = cms.Sequence(type0PFMEtCorrectionPFCandToVertexAssociationForValidationMiniAOD * JetValidationMiniAOD * METValidationMiniAOD)
+validationMiniAOD = cms.Sequence(type0PFMEtCorrectionPFCandToVertexAssociationForValidationMiniAOD * JetValidationMiniAOD * METValidationMiniAOD * tauValidationSequenceMiniAOD)
```
Comment (jfernan2) [inline Configuration/StandardSequences/python/Validation_cff.py]: in order to keep uniformity, couldn't the word Sequence be removed from tauValidationSequenceMiniAOD?

Comment (aspiezia) [reply Configuration/StandardSequences/python/Validation_cff.py]: @jfernan2 
Unfortunately this change will be in conflict with:
https://github.com/cms-sw/cmssw/blob/d15737307b0e7b710c075f0e9303e9aeee169470/Validation/RecoTau/python/RecoTauValidation_cff.py#L3-L4

**Comment by jfernan2** [comment]
@aspiezia this PR is adding 84 bew recoTauV plots but not in all workflows involving miniAOD:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_11_0_X_2019-09-03-2300+288469/33358/dqm-histo-comparison-summary.html

In particular 136.7611 and 136.8311 which have reminiAOD do not produce the plots. Can you confirm the behaviour is the desired? Thanks

**Comment by aspiezia** [comment]
Dear @jfernan2 ,
are 136.7611 and 136.8311 the only workflows involving miniAOD that do not have the histograms?
Could you please give me the cmsDriver.py commands used for this workflows, so that I can test them?
Thanks,
Aniello

**Comment by jfernan2** [comment]
@aspiezia not only those two I am afraid, you can check the matrix and the corresponding configurations in:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-288469/2356/runTheMatrix-results/
clicking in any of the folders and the cmdLog file

**Comment by aspiezia** [comment]
Dear @jfernan2 
the reason why in those workflows the new histograms are not present is because in the cmsDriver.py command they have only the option "-s PAT,DQM:@miniAODDQM", while the new histograms appear when also the option "VALIDATION:@standardValidation+@miniAODValidation" is used.
I have implemented them following what was done for the miniAOD from other groups, for example: https://tinyurl.com/y5h3nn7b

Does it make sense to you?
Aniello

**Comment by jfernan2** [comment]
@aspiezia yes, thanks!

**Comment by jfernan2** [comment]
+1

**Comment by fabiocos** [comment]
+operations

the update of StandardSequences looks coherent with the purpose of the PR

**Comment by fabiocos** [comment]
+1

**Pull request merged**